webpack/webpack.common.js

```diff
@@ -32,6 +32,14 @@ plugins:[
     {
       template: './src/index.html'
     }
-  )
+  ),
+  new StyleLintPlugin({
+    configFile: '.stylelintrc',
+    context: 'src',
+    files: '**/*.css',
+    failOnError: false,
```

Comment:
This is false by default, we can remove it.

Reply:
ok. will change to true